addons/website/models/website.py

```diff
@@ -857,6 +857,11 @@ def search_pages(self, needle=None, limit=None):
                 break
         return res
 
+    def get_suggested_controllers(self):
+        """ Returns a list of url. """
+        suggested_controllers = ['/']
```

Comment:
what's do you think about a tuple idea: (_('Ecommerce'), '/shop'),

Probably more user friendly no ?